docs/buildtest_perlmutter.rst

```diff
@@ -16,18 +16,22 @@ To get started please load the **python** module since you will need python 3.7
 
     module load python
 
-Next, you should :ref:`Install buildtest <installing_buildtest>` by cloning the repository in your $HOME directory.
+Next, you should :ref:`Install buildtest <installing_buildtest>` by cloning the repository into your home directory::
 
-Once you have buildtest setup, please clone the following repository https://github.com/buildtesters/buildtest-nersc in your $HOME directory as follows::
+    git clone https://github.com/buildtesters/buildtest.git
```

Comment:
maybe put cd $HOME here as well since you're doing this type of directory shifting before other commands